CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Add strategy for American Government answer key (minor)
 * Add `BakeReferences` direction (minor)
 * Fix xmlns string replacement done in PR #209 (minor)
+* Remove depreciated directions `BakeNotes` and `BakeExercises` (minor)
```

Comment:
This needs to be major b/c older recipes (e.g. Chemistry) may still rely on them.  In semantic versioning, minor changes are for new features or changes that don't have externally visible impacts